torchaudio/models/_wavernn.py

```diff
@@ -192,3 +193,125 @@ def forward(self, specgram: Tensor) -> Tensor:
         upsampling_output = upsampling_output.squeeze(1)[:, :, self.indent:-self.indent]
 
         return upsampling_output, resnet_output
+
+
+class _WaveRNN(nn.Module):
+    r"""WaveRNN model based on "Efficient Neural Audio Synthesis".
+
+    The paper link is https://arxiv.org/pdf/1802.08435.pdf. The input channels of waveform
```

Comment:
nit: can we follow [this](https://github.com/pytorch/audio/blob/master/torchaudio/transforms.py#L512) example for the link?

Reply:
Updated.